packages/react-router/src/Matches.tsx

```diff
@@ -136,13 +136,13 @@ export type MakeRouteMatchFromRoute<TRoute extends AnyRoute> = RouteMatch<
 >
 
 export interface RouteMatch<
-  TRouteId,
-  TFullPath,
-  TAllParams,
-  TFullSearchSchema,
-  TLoaderData,
-  TAllContext,
-  TLoaderDeps,
+  out TRouteId,
```

Comment:
Missed these I guess but they were showing up in the profiler with the variance worker